dacbarbos/README.md

```diff
@@ -0,0 +1,7 @@
+# W3ID for /dacbarbos/
+
+## Workspace: DACBARBOS Brand's permanent ID
+
+## Contacts: DACBARBOS Brand's Organization Administrator (admin@dacbarbos.eu)
+
+### GitHub username: @orgadm
```

Comment:
```suggestion
## Workspace

DACBARBOS Brand's permanent ID

## Contacts

DACBARBOS Brand's Organization Administrator (admin@dacbarbos.eu)
GitHub username: @orgadm
```

Reply:
PR #4621 is a follow-up to apply the suggested change.